release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -58,14 +58,10 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-15-installation-and-update"]
 === Installation and update
 
-The Cluster API Provider for OpenStack (CAPO) is now deployed by default when the `TechPreviewNoUpgrade` feature flag is enabled. The lifecycle of this component is managed by the CAPI Operator. When the `TechPreviewNoUpgrade` feature flag is enabled, an OpenStack cluster and additional OpenShift cluster are automatically created for the current OpenShift cluster. It is now possible to configure the CAPI machine and OpenStack machine resources in a similar fashion to how the Mail Application Programming Interface (MAPI) resources are configured. It is important to note that these resources are equivalent but not identical.
```

Comment:
Ok, so it took me a minute to follow the breadcrumbs. I _think_ what you want to do is add `[id="ocp-4-15-CAPO-tech-preview-no-upgrade"]
== CAPO integration into the cluster CAPI Operator(Tech Preview)` to line 60 of the RN file (i.e. this file). 

You then need to put back line 63-68. I'm guessing you had a merge conflict and just picked the wrong content. If you need help sorting this ping me @obrown1205

Reply:
Thanks for spotting this @JoeAldinger. Yes as it stands this PR would remove my release note for user-managed encryption.

Reply:
Thank you!